### ISSUE TITLE
Deployment Types

### ISSUE THREAD

**Issue by RichiCoder1** [aws/apprunner-roadmap#4]
**Community Note**
* Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do * not help prioritize the request
If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
In a nutshell, the equivalent of deployment types with ECS: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/deployment-types.html

**Describe alternatives you've considered**
Doing traffic shifting at a higher layer (LB).

**Comment by evankanderson** [comment]
Selfishly, I'd vote for percentage-based (LB) deployments if you already have autoscaling, possibly with automated gradual shift based on application scale (e.g. > 1000 instances, no more than 10% per minute).
